packages/analytics/e2e/analytics.e2e.js

```diff
@@ -1133,5 +1134,28 @@ describe('analytics() modular', function () {
         await setConsent(getAnalytics(), consentSettings);
       });
     });
+
+    describe('getGoogleAnalyticsClientId()', function () {
+      it('Error when trying to use getGoogleAnalyticsClientId() on non-web platforms', async function () {
+        if (!Platform.other) {
+          try {
+            const { getAnalytics } = analyticsModular;
+            await firebase.analytics.getGoogleAnalyticsClientId(getAnalytics());
+            fail('Should have thrown an error');
+          } catch (e) {
+            e.message.should.equal('getGoogleAnalyticsClientId is web-only.');
+          }
+        }
+        this.skip();
+      });
+
+      it('A response is received from getGoogleAnalyticsClientId() on web platforms', async function () {
+        const { getAnalytics } = analyticsModular;
+        if (Platform.OS === 'web') {
+          await firebase.analytics.getGoogleAnalyticsClientId(getAnalytics());
```

Comment:
these follow a pattern that we can + should assert against. Not down to a really really specific level like lengths but here's the format:

https://github.com/invertase/react-native-firebase/blob/a909d1beb597dcb92035a2ee6e66995c3aee8a5a/packages/analytics/lib/web/api.js#L36

Should be a string in general, more specifically is string.string, and then even more specifically is random-number-as-string, period, timestamp as string

I think just asserting it is a non-zero length string is sufficient, but is much stronger as a test then just checking it didn't error